src/app/constant.js

```diff
@@ -356,6 +356,24 @@ export const API_BASE_URLS = {
   CRS: '/crs/'  //Example /crs/<epsg_code>
 };
 
+/**
+ * LOCAL ITEM IDS
+ * Used to store id used by application
+ * Each item has:
+ * {
+ *  id: unique identifier,
+ *  value: initial value
+ *  }
+ * @since v3.8
+ */
+
```

Comment:
```suggestion
/**
 * @typedef LocalItem
 * @property {string} id unique identifier
 * @property {any} value initial value
 */

/**
 * LOCAL ITEM IDS - used to store id used by application
 * 
 * @type {Object<string, LocalItem>}
 * 
 * @since v3.8
 */
```